### PR TITLE
python-uci: add package

### PR DESCRIPTION
**Pull request by whooo**
Python bindings for libuci, useful for writing ansible modules to manage OpenWRT systems.
Reused Makefile from upstream: https://gitlab.nic.cz/turris/turris-os-packages/-/blob/develop/lang/python-uci/Makefile

Signed-off-by: Erik Larsson <who+github@cnackers.org>

Maintainer:  @BKPepe 
Compile tested: ipq40xx, EnGenius EAP1300, snapshot
Run tested: ipq40xx, EnGenius EAP1300, snapshot, tested with using the uci module in an ansible module

Description:
python-uci is a python module with bindings towards libuci from the Turris project.
I use it for ansible modules, but there is probably other use-cases as well.

### CODE REVIEW

**Comment by BKPepe** [comment]
Could you please use the latest version of Makefile from develop branch here: https://gitlab.nic.cz/turris/turris-os-packages/-/blob/develop/lang/python-uci/Makefile ?

**Comment by whooo** [comment]
> Could you please use the latest version of Makefile from develop branch here: https://gitlab.nic.cz/turris/turris-os-packages/-/blob/develop/lang/python-uci/Makefile ?

Sure, I'll do that.
Do you want me to leave NIC.CZ as maintainer? It doesn't really matter for, I mostly want the package in OpenWRT

File: lang/python/python-uci/Makefile
Comment (BKPepe) [inline lang/python/python-uci/Makefile:23]:
```suggestion
include ../pypi.mk
include $(INCLUDE_DIR)/package.mk
include ../python3-package.mk
```

File: lang/python/python-uci/Makefile
Comment (BKPepe) [inline lang/python/python-uci/Makefile:17]: In OpenWrt, it is required to use real name and has email address connected to GitHub acc. So, you can be maintainer or put me as a maintainer.

```suggestion
PKG_MAINTAINER:=Josef Schlehofer <josef.schlehofer@nic.cz>
```

File: lang/python/python-uci/Makefile
Comment (BKPepe) [inline lang/python/python-uci/Makefile:12]: This package was not present in OpenWrt packages feed before, so we should start with 1, but there was a recent change that there is a possibility to use autorelease.

```suggestion
PKG_RELEASE:=$(AUTORELEASE)
```

**Review by BKPepe — COMMENTED** [review]
Just small things, which should be considered or fixed before merging. Otherwise 👍

File: lang/python/python-uci/Makefile
Comment (BKPepe) [inline lang/python/python-uci/Makefile:32]: Sorry! I missed this thing. This is no longer necessary. This is used in OpenWrt 19.07 version, which has two version of Pythons and in OpenWrt master, there is only one, so this can be removed.

```suggestion
```

**Comment by whooo** [comment]
Had to to do an amended commit to make the DCO check happy

**Review by BKPepe — APPROVED** [review]

**Pull request merged**
